tests/GoogleFontsTest.php

```diff
@@ -34,7 +34,7 @@
     config()->set('google-fonts.fonts', ['cow' => 'moo']);
     config()->set('google-fonts.fallback', true);
 
-    $fonts = app(GoogleFonts::class)->load('cow', forceDownload: true);
+    $fonts = app(GoogleFonts::class)->load(['font' => 'cow'], forceDownload: true);
```

Comment:
Could you use [a Pest dataset here](https://pestphp.com/docs/datasets), to run the test with both the array and the string?